tests/integration_test/services_test.py

```
@@ -5051,9 +5049,12 @@ def test_catalog_sync_item_toggle_ai_disabled_meta_disabled(mock_embedding, mock
     assert actual["data"] is None
     assert actual["success"]
 
+    latest_log = CatalogSyncLogs.objects(bot=str(pytest.bot)).order_by("-start_timestamp").first()
+    log_id = str(latest_log.id)
+
     complete_end_to_end_event_execution(
         pytest.bot, "integration@demo.ai", EventClass.catalog_integration, sync_type="item_toggle", token=token,
-        provider="petpooja", data=item_toggle_payload
+        provider="petpooja", log_id = log_id
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)